src/common/_modules/private_endpoint/data.tf

```diff
@@ -0,0 +1,12 @@
+# CGN PostgreSQL Single Server
+
+data "azurerm_private_dns_zone" "cgn_psql_private_dns_zone" {
+  name                = "privatelink.postgres.database.azure.com"
+  resource_group_name = "io-p-rg-common"
```

Comment:
I think we can remove this data by getting the output as suggested in the following comment